README.md

```diff
@@ -54,7 +54,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.4/install.html#
 | | `bbb_turn_enable` | enable the use uf TURN in general | `yes` | |
 | | `bbb_stun_servers` | a list of STUN-Server to use | `{{ bbb_hostname }}` | an array with key `server` - take a look in defaults/main.yml |
 | | `bbb_ice_servers` | a list of RemoteIceCandidate for STUN | `[]` | in array with key `server` |
-| | `bbb_turn_servers` | a list of TURN-Server to use | `{}` | take a look in defaults/main.yml |
+| | `bbb_force_install` | install even if meetings are running | `no` | current meetings will be terminated |
```

Comment:
Just preference discussion.
`bbb_force_install` may read to an unknown user that somehow an installation gets forced.
Wouldn't it make more sense to call this variable `bbb_ignore_running_meetings`?